src/scratch.ts

```diff
@@ -1,20 +1,12 @@
-import { Chess } from 'chess.js';
-import { gameChunks } from './fileReader';
+import { getMoveDistance } from './metrics';
 
-// Some scratch code to test the chess.js library
-const board = new Chess(
-  'rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR w KQkq - 0 1'
-);
-console.log(board.ascii());
-const a = board.get('e2');
-console.log(a);
+async function main() {
+  try {
+    const result = await getMoveDistance('data/lichess_db_standard_rated_2013-01_TEST_SET.txt');
+    console.log(result);
+  } catch (error) {
+    console.error(error);
+  }
+}
 
-// Demo the generator
-const gen = gameChunks('data/lichess_db_standard_rated_2013-01.pgn');
-
-gen.next().then((x) => {
-  console.log(x);
-});
-gen.next().then((x) => {
-  console.log(x);
-});
+main().then(() => {});
```

Comment:
I think we can jsut delete this file, made it to start with but we should formalize and use unit tests